src/Components/AddTask.jsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import Card from "./Card.jsx";
+import PrimaryButton from "./PrimaryButton.jsx";
+import "./AddTask.css";
+import Close from "./Svg/close.svg";
+function AddTask(props) {
```

Comment:
```suggestion
function AddTask({trigger}) {
```
Destructing props can make code less verbose and more readable, If you want to learn more, read [this medium article by Lindsay Criswell](https://medium.com/@lcriswell/destructuring-props-in-react-b1c295005ce0)

Reply:
```suggestion
const AddTask = (props) => {
```
For Consistency: Strictly Follow ES6 Arrow Function Syntax throughout